### PR TITLE
Fix display of large components

### PR DESCRIPTION
**Pull request by su-narthur**
Add CSS to allow for oversized components to scroll in all directions. See [issue #1225](https://github.com/storybooks/storybook/issues/1225#issuecomment-307259951).

Issue: Overflowing portions of large components are inaccessible when using Centered plugin.

## What I did

I added `overflow: auto` to the outer wrapper and `margin: auto` to a new inner wrapper.

## How to test

Create a component larger than your viewport and check that you can stroll both up and down to see the entire component.

### CODE REVIEW

File: addons/centered/src/index.js [hunk 1]
```diff
@@ -9,8 +9,13 @@ const style = {
...
+  overflow: 'auto',
 };
 
+const innerStyle = {
+  margin: 'auto',
+}
+
 export default function(storyFn) {
-  return <div style={style}>{storyFn()}</div>;
+  return <div style={style}><div style={innerStyle}>{storyFn()}</div></div>;
```
Comment (ndelangen) [inline addons/centered/src/index.js]: I think adding another container defeats the purpose of the centered addon, I'll have to test.

Comment (su-narthur) [inline addons/centered/src/index.js]: Were you able to test this? I'd be happy to try to improve the fix if this proves to have problems.

Comment (ndelangen) [inline addons/centered/src/index.js]: I'll test this

**Comment by ndelangen** [comment]
It works wonders! Thanks you, you are awesome @su-narthur!

Sorry it took so long to view and merge, totally my fault!

**Review by ndelangen — APPROVED** [review]

**Pull request merged**